README.rst

```diff
@@ -137,11 +137,11 @@ Updating the prebuilt jar
 
 The prebuilt jar is currently generated by hand, whenever the code seems stable and has enough new features to warrant the effort.
 
-To update the jar, we use a different set of maven commands:
+To update the jar, we use a different set of maven commands::
```

Comment:
yes, its in [reStructured text](http://sphinx-doc.org/rest.html) format

you can see that the format is wrong at the moment here https://github.com/googlegenomics/dataflow-java/blob/master/README.rst#updating-the-prebuilt-jar and this fixes it